### DISCUSSION TITLE
remote-config - Warning "remoteConfig().getValue(*).value has been removed" with getAll()

### DISCUSSION THREAD

**Discussion by tonyAlgara** [invertase/react-native-firebase#4190] [Q&A] [answered]
Hello, since last update we are getting a warning using the method `firebase.remoteConfig().getAll();`

`WARN     firebase.remoteConfig().getValue(*).value has been removed. Please use one of the alternative methods such as firebase.remoteConfig().getValue(*).asString()`

We managed to work but wonder if getAll() method needs and update.

EDIT solution: Issue isn't in getAll() 
from docs: 
`Object.entries(parameters).forEach(([key, parameter]) => {
  console.log('Key: ', key);
  console.log('Value: ', parameter.value);
});`

instead of .value should be asString, asBoolean, etc

Thanks

**Comment by mikehardy** [comment, marked as answer]
The release notes will always contain information about the breaking changes - 

https://github.com/invertase/react-native-firebase/blob/master/packages/remote-config/CHANGELOG.md#800-2020-07-23

**Reply by aluminium22** [reply]
I'm experiencing this same error. (I'm migrating from v5 to v6 of react-native-firebase). I can't find anywhere in my code that I'm using firebase.remoteConfig.getValue(*).value. am I missing something? 
I followed the migration steps, then went through each of the packages that I needed to install and followed their steps.
I'll attach a screenshot of my error below.
<img width="436" alt="Screen Shot 2021-06-08 at 8 58 23 AM" src="https://user-images.githubusercontent.com/6405831/121189554-3520d500-c838-11eb-8a22-821411a237c8.png">

**Reply by mikehardy** [reply]
@aluminium22 with apologies, your basic assumption is flawed. You assert you are not using it despite the evidence of the very screenshot you post. You are using it somewhere somehow. Perhaps caching unexpectedly (`npx react-native-clean-project`) or something else. But you are using it somehow. Find the usage and remove it.

**Reply by aluminium22** [reply]
Thank you for the super quick response. I had assumed I had something put together incorrectly, and your response helps clarify this. I wanted to make sure this wasn't a common error in migration, so thank you. I'll keep searching for the culprit.
